unity/Assets/Scripts/PhysicsRemoteFPSAgentController.cs

```diff
@@ -203,12 +207,43 @@ public void SetDecayTemperatureBool(ServerAction action)
 
         // Update is called once per frame
         void Update() {
-            if (FlightMode) {
-                if (thrust.magnitude > 0.1) {
-                    m_CharacterController.Move(thrust * Time.deltaTime);
+
+        }
+
+        void FixedUpdate(){
```

Comment:
you could override this and simplify the logic, and have each controller decide what fixed update

Reply:
agreed, I will start working on a drone controller that inherits from PhysicsRemoteFPSAgentController and see where that goes